### PR TITLE
Added new check for deadqueue

### PR DESCRIPTION
**Pull request by bennythejudge**
- updated README
- added new script to check that the dead queue is empty

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,21 @@
 
 ## Functionality
 
-Check varius metrics and checks for Sidekiq. Requires [sidekiq-monitor-stats](https://github.com/harvesthq/sidekiq-monitor-stats)
+Check varius metrics and checks for Sidekiq.
+The following scripts:
+
+ * bin/check-sidekiq.rb
+ * bin/sidekiq-metris.rb
```
Comment (analytically) [inline README.md]: sidekiq-metrics.rb

**Comment by eheydrick** [comment]
Thanks @bennythejudge! Can you update the CHANGELOG.

**Comment by bennythejudge** [comment]
CHANGELOG updated but I have not added a tag (yet). Normally I'd do that after the PR is merged. I hope it's ok. Thanks.

**Comment by eheydrick** [comment]
Thanks. Usually the new entries would go under the unreleased section but no big deal.

**Comment by bennythejudge** [comment]
Thanks!! I actually thought so. Should I (can I?) propose the new version/tag?

**Pull request merged**